ursula_cli/shell.py

```diff
@@ -389,9 +390,9 @@ def _run_heat(args, hot):
         while not test_ssh(floating_ip, user, ssh_key_path):
             LOG.debug("waiting for SSH connectivity...")
             time.sleep(5)
-    else:
```

Comment:
test_ssh so not loop indefinitely. We should have timeout and give up after 2 mins.